docs/creating-new/create/cmake.rst

```diff
@@ -343,25 +343,25 @@ Add ``hunter_config`` directive with default version to
 Create example
 ==============
 
-Simple example will be used to test integration of package. Copy template
-example and substitute all strings ``foo`` with ``hunter_box_1``:
+ To test the integration of the package into Hunter a simple example will be used.
```

Comment:
> the integration of the package **into Hunter**

I don't think it's correct. We are not integrating package into Hunter. Question about integrating package into **another project**. That's what we are planning to test.